packages/arduino_ide.rb

```diff
@@ -40,26 +46,21 @@ def self.build
       system "echo '#!/bin/bash' > arduino"
       system "echo >> arduino"
       system "echo 'echo \"Enabling Arduino write access...\"' >> arduino"
-      system "echo 'sudo  chmod o+w /dev/ttyACM*' >> arduino"
-      system "echo 'sommelier -X --x-display=\$DISPLAY --scale=\$SCALE #{CREW_PREFIX}/share/arduino-#{version}/arduino \"$@\"' >> arduino"
+      system "echo 'sudo chmod o+rw /dev/ttyACM*' >> arduino"
+      system "echo '#{CREW_PREFIX}/share/arduino-#{version}/arduino \"$@\"' >> arduino"
       system "echo 'echo \"Disabling Arduino write access...\"' >> arduino"
-      system "echo 'sudo chmod o-w /dev/ttyACM*' >> arduino"
+      system "echo 'sudo chmod o-rw /dev/ttyACM*' >> arduino"
     end
   end
 
   def self.install
-    case ARCH
-    when 'x86_64'
-      platform = 'linux64'
-    when 'i686'
-      platform = 'linux32'
-    when 'armv7l', 'aarch64'
-      platform = 'linuxarm'
-    end
+    version = @version
+    platform = @platform
     Dir.chdir("build") do
       FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share"
-      system "tar", "xpvf", "linux/arduino-#{version}-#{platform}.tar.xz", "-C", "#{CREW_DEST_PREFIX}/share/"
+      system "tar", "-xpf", "linux/arduino-#{version}-#{platform}.tar.xz", "-C", "#{CREW_DEST_PREFIX}/share/"
```

Comment:
The dash is not necessary on the tar command.

Reply:
And why did you remove the verbose option?

Reply:
Done. Did not add back `-v` option as not needed.